src/main/java/com/team2357/lib/subsystems/LimelightSubsystem.java

```diff
@@ -19,10 +32,11 @@ public class LimelightSubsystem extends ClosedLoopSubsystem {
 
   public static class Configuration {
 
-    public int m_humanPipelineIndex = 0;
+    public int m_humanPipelineIndex = 2;
```

Comment:
This value should probably be defined in Constants rather than here.

Reply:
This is in a config class, so we can always just override it. Do we want a specific constants file in robotlib and get rid of the configuration class?

Reply:
In this case, we need to coordinate Limelight config with this config. For anything that has external dependencies like that, I think we want it in one file. It wouldn't be fun to have to dig to find where this is set.